packages/workbox-sw/controllers/WorkboxSW.mjs

```diff
@@ -28,6 +28,7 @@ const MODULE_KEY_TO_NAME_MAPPING = {
   strategies: 'strategies',
   precaching: 'precaching',
   routing: 'routing',
+  cacheableResponse: 'cacheable-response',
```

Comment:
FWIW, I realize that `broadcastCacheUpdate` is missing from there, too. Do you want to sneak it in as part of this PR?

Reply:
Nah I'll do it along with a PR for the broadcast cache update demo